src/faebryk/libs/kicad/fileformats.py

```diff
@@ -705,13 +710,15 @@ class C_rotate:
 
 @dataclass
 class C_kicad_pcb_file(SEXP_File):
+    @dataclass_json(undefined=Undefined.INCLUDE)
```

Comment:
Am I understanding correctly that we are abusing the dataclass_json decorator everywhere for the catch all?